app/src/main/java/com/github/sdp_begreen/begreen/fragments/SendPostFragment.kt

```diff
@@ -140,10 +153,11 @@ class SendPostFragment : Fragment() {
      */
     companion object {
         @JvmStatic
-        fun newInstance(param1: String) =
+        fun newInstance(param1: String, param2: Boolean = false) =
```

Comment:
I would maybe give a name that explains a little bit more what the parameter is supposed to represent, rather than `param1` and `param2`.

Reply:
Totally agree, thanks